server/application/application.go

```diff
@@ -1366,6 +1366,35 @@ func (s *Server) RevisionMetadata(ctx context.Context, q *application.RevisionMe
 	})
 }
 
+func (s *Server) RevisionChartDetails(ctx context.Context, q *application.RevisionMetadataQuery) (*appv1.ChartDetails, error) {
```

Comment:
Please add godoc for public functions

Reply:
added, github UI doesn't show it for some reason